core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -2040,7 +2040,7 @@ sealed trait ZIO[-R, +E, +A] extends Serializable with ZIOPlatformSpecific[R, E,
    */
   final def scheduleBackground[R1 <: R, B](schedule: => Schedule[R1, Any, B])(implicit
     trace: ZTraceElement
-  ): ZIO[R1 with Clock with Scope, E, B] =
+  ): ZIO[R1 with Clock with Scope, E, Fiber[Any, B]] =
```

Comment:
You can delete the type parameter `B` and use `Any` for that slot in `Schedule`.